mkat_fpga_tests/test_cbf.py

```diff
@@ -1470,3 +1470,51 @@ def test_all_delays(self):
             Aqf.less(abs_diff, 1,'Check that the maximum degree between '
                 'expected and actual phase difference between intergrations '
                 'is below 1 degree: {0:.3f} degree\n'.format(abs_diff))
+    @aqf_vr('TP.C.1.17')
+    def test_config_report(self):
+        """CBF Report configuration"""
+
+        repos = ['corr2', 'mkat_fpga', 'casperfpga', 'mkat_fpga_tests',
```

Comment:
Are you also running the tests from mkat_fpga_tests? 

Perhaps you could go one better for the python modules and import each one and the check it's __file__ to see what directory it is installed in? But we can merge as is (don't push such changes to this branch), just a suggestion for future work.